lisp/ess-custom.el

```diff
@@ -2896,45 +2901,87 @@ the variable `ess-help-own-frame' is non-nil."
   :type 'alist)
 
 
- ; User changeable variables
+ ; Faces
 ;;;=====================================================
-;;; Users note: Variables with document strings starting
-;;; with a * are the ones you can generally change safely, and
-;;; may have to upon occasion.
-
-(defvar ess-function-call-face 'ess-function-call-face
-  "Face name to use for highlighting function calls.")
-
-(defvar ess-numbers-face 'ess-numbers-face
-  "Face name to use for highlighting numbers.")
-
-(defvar ess-%op%-face 'ess-%op%-face
-  "Face name to use for highlighting %op% operators.")
 
+(defconst ess-function-call-face 'ess-function-call-face)
 (defface ess-function-call-face
   '((default (:slant normal :inherit font-lock-function-name-face)))
   "Font Lock face used to highlight function calls in ess buffers."
-  :group 'ess)
+  :group 'ess-faces)
 
+(defconst ess-numbers-face 'ess-numbers-face)
 (defface ess-numbers-face
   '((default (:slant normal :inherit font-lock-type-face)))
```

Comment:
Why change `slant`?

Reply:
Orh. Nevermind, that's our default for whatever reason. You didn't introduce any change here.